noaa/challenges/default.json

```diff
@@ -226,4 +226,68 @@
         },
         {% include 'aggs-tasks.json' %}
       ]
+    },
+    {
+      "name": "filter-aggs",
+      "description": "Checks the performance of many aggregations",
+      "default": false,
+      "schedule": [
+        {
+          "operation": "delete-index"
+        },
+        {
+          "operation": {
+            "operation-type": "create-index",
+            "settings": {{index_settings | default({}) | tojson}}
+          }
+        },
+        {
+          "name": "check-cluster-health",
+          "operation": {
+            "operation-type": "cluster-health",
+            "index": "weather-data-2016",
+            "request-params": {
+              "wait_for_status": "{{cluster_health | default('green')}}",
+              "wait_for_no_relocating_shards": "true"
+            },
+            "retry-until-success": true
+          }
+        },
+        {
+          "operation": "index",
+          "#COMMENT": "This is an incredibly short warmup time period but it is necessary to get also measurement samples. As this benchmark is rather about search than indexing this is ok.",
+          "warmup-time-period": 10,
+          "clients": {{bulk_indexing_clients | default(8)}},
+          "ignore-response-error-level": "{{error_level | default('non-fatal')}}"
+        },
+        {
+          "name": "refresh-after-index",
+          "operation": "refresh",
+          "clients": 1
+        },
+        {
+          "operation": "force-merge",
+          "clients": 1{%- if max_num_segments is defined %},
+          "max-num-segments": {{max_num_segments}}
+           {%- endif %}
+        },
+        {
+          "name": "refresh-after-force-merge",
+          "operation": "refresh",
+          "clients": 1
+        },
+        {% include 'filters-aggs-tasks.json' %}
+        {
+          "operation": "filter-agg-with-high-cardinality-filter",
```

Comment:
is there a reason why the two operations `filter-agg-with-high-cardinality-filter` and `filter-per-10-degrees-interval` don't live within the included `filters-aggs-tasks.json`?

Reply:
These two operations test run the `filter` aggregation and the aggregations in the `filters-aggs-tasks.json` file run the `filters` aggregation. The refactoring in elastic/elasticsearch#90318 changes the implementation of `filter` aggregation to use the implementation of the `filters` aggregation. And `filter` aggregations defined here exist just to double check we got the expected performance improvement and these `filter` aggregation don't need to run highly with all the other aggregations.

Initially the idea was to run `filters` aggregation as part of the `aggs` challenge, but we should first take a look at all the other operations that run as part of the `aggs` challenge. As the number of operations make investigating performance regressions more difficult.